pipelines/release-images-cf-deployment/tasks/build.sh

```diff
@@ -16,16 +16,16 @@ start_docker \
 trap 'stop_docker' EXIT
 
 # Login to the Docker registry.
-echo "${DOCKER_TEAM_PASSWORD_RW}" | docker login --username "${DOCKER_TEAM_USERNAME}" --password-stdin
+echo "${DOCKER_TEAM_PASSWORD_RW}" | docker login "${DOCKER_REGISTRY}" --username "${DOCKER_TEAM_USERNAME}" --password-stdin
 
 # Extract the fissile binary.
 tar xvf s3.fissile-linux/fissile-*.tgz --directory "/usr/local/bin/"
 
 # Pull the stemcell image.
-stemcell_version="$(cat s3.stemcell-version/*-version)"
+stemcell_version="$(cat s3.stemcell-version/${STEMCELL_VERSIONED_FILE##*/})"
```

Comment:
can you double quote `STEMCELL_VERSIONED_FILE##*/` to prevent globbing?

Reply:
DONE!